.github/workflows/py-tests.yml

```diff
@@ -33,11 +33,13 @@ permissions:
 
 jobs:
   py-run-tests:
-    runs-on: ubuntu-latest
+    #runs-on: ubuntu-latest
+    runs-on: mgmt-x86
     strategy:
       fail-fast: false
       matrix:
-        py-version: ['3.8', '3.9', '3.10', '3.11']
+        #py-version: ['3.8', '3.9', '3.10', '3.11']
+        py-version: ['3.10']
```

Comment:
Are we going to run the tests only on 3.10 from now on?

Reply:
No, it was just to test on our private runners one build at a time. Given RYUK is not deleting the dangling images I think this is not going to solve the disk usage issues we are facing